xla/service/gpu/ir_emitter_triton_test.cc

```diff
@@ -5641,6 +5657,27 @@ CHECK: wgmma
 )");
 }
 
+// Test usage of default matmul config information.
+TEST_F(TritonGemmTest, TestNoAutotuner) {
```

Comment:
Thanks for adding the test! Unfortunately, it's not clear that this is testing the path we care about---nothing tells us that things were lowered through to Triton here.

Could you please also add a `MatchOptimizedHlo` that checks the relevant properties (that all the relevant parameters in the backend config are set) to make sure that the right path is exercised?

Reply:
Modified the test case.
The issue is present only on rocm because GemmFusionAutotuner pass is not added into pipeline at all.
For Cuda, even with xla_gpu_autotune_level set to 0, it will always provide some config, thus check will never fail.
That is the reason why test is skipped for cuda.